Tests/govuk_ios/govuk_ios_unit_tests/Specs/Services/SignableDecoderTests.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import Testing
+@testable import govuk_ios
+
+struct SignableDecoderTests {
+
+    @Test
+    func test_verifyValidSignature() async throws {
+        let appConfigData = try #require(getMockConfigData(fileName: "MockAppConfigResponse"), "Unable to read mock data")
+        let appConfig = try #require(try? SignableDecoder().decode(AppConfig.self, from: appConfigData), "Unable to decode AppConfig")
+        #expect(appConfig != nil)
+    }
+    
+    @Test
+    func test_verifyInValidSignature() async throws {
```

Comment:
Can you add some expectations to the names please?
`<action>_<condition(s)>_<expectation>` is the structure we are following for test names.

Reply:
Done